File: spring-cloud-father-new-producer0/src/main/java/com/ciel/springcloudfathernewproducer0/controller/MainController.java
package com.ciel.springcloudfathernewproducer0.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;

@RestController
public class MainController {

    @GetMapping("/")
    public Object index(){
        return Map.of("MSG","通过其他微服务调用");
    }
}

File: spring-cloud-new-father-kotlin/src/main/kotlin/com/ciel/springcloudnewfatherkotlin/controller/MainController.kt
package com.ciel.springcloudnewfatherkotlin.controller

import com.ciel.springcloudnewfatherkotlin.service.UserService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/main")
class MainController {

    @Autowired
    lateinit var user : UserService;

    @GetMapping("/main")
    fun main(): Any {
        return user.userInfo()
    }



}